dashboards/Hub-ECS.json

```diff
@@ -1339,6 +1339,193 @@
       "transform": "table",
       "type": "table"
     },
+    {
+      "aliasColors": {},
+      "bars": false,
+      "dashLength": 10,
+      "dashes": false,
+      "datasource": "$Prometheus",
+      "fill": 1,
+      "gridPos": {
+        "h": 9,
+        "w": 12,
+        "x": 0,
+        "y": 30
+      },
+      "id": 25,
+      "legend": {
+        "avg": false,
+        "current": false,
+        "max": false,
+        "min": false,
+        "show": true,
+        "total": false,
+        "values": false
+      },
+      "lines": true,
+      "linewidth": 1,
+      "links": [],
+      "nullPointMode": "null",
+      "percentage": false,
+      "pointradius": 5,
+      "points": false,
+      "renderer": "flot",
+      "seriesOverrides": [],
+      "spaceLength": 10,
+      "stack": false,
+      "steppedLine": false,
+      "targets": [
+        {
+          "expr": "sum(uk_gov_ida_hub_samlproxy_resources_SamlMessageReceiverApi_handleRequestPost_2xx_responses_total) / (sum(uk_gov_ida_hub_samlproxy_resources_SamlMessageReceiverApi_handleRequestPost_1xx_responses_total + uk_gov_ida_hub_samlproxy_resources_SamlMessageReceiverApi_handleRequestPost_2xx_responses_total + uk_gov_ida_hub_samlproxy_resources_SamlMessageReceiverApi_handleRequestPost_3xx_responses_total +\nuk_gov_ida_hub_samlproxy_resources_SamlMessageReceiverApi_handleRequestPost_4xx_responses_total +\nuk_gov_ida_hub_samlproxy_resources_SamlMessageReceiverApi_handleRequestPost_5xx_responses_total))",
```

Comment:
you need a `rate` function on your counters here https://www.robustperception.io/rate-then-sum-never-sum-then-rate

You're computing y/x but you want dy/dx

Reply:
I also had a separate conversation with Andy about how this is pointless for SSP
and if we have the 5xxs for everything (except frontend 🙈 ) then this PR is pointless